charts/consul/templates/connect-inject-authmethod-clusterrolebinding.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ template "consul.fullname" . }}-connect-injector-authmethod-authdelegator-role-binding
+  name: {{ template "consul.fullname" . }}-connect-injector-authmethod-authdelegator
```

Comment:
this can be `-connect-injector-authdelegator` since this is a special clusterrolebinding where we're binding to a preset role vs how we're usually binding to a role we've created. So this can be with the special name and the rest can be just the name of the component, i.e. `-connect-injector`